build/yaml/webhook/nsx-operator.yaml

```diff
@@ -0,0 +1,510 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: nsx-operator
+  namespace: vmware-system-nsx
+  labels:
+    component: nsx-operator
+spec:
+  ports:
+    - name: metrics
+      port: 8093
+      targetPort: 8093
+      protocol: TCP
+  selector:
+    component: nsx-operator
+
+---
+
+# Create Namespace for NSX owned resources
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: vmware-system-nsx
+
+
+---
+
+# Create a ServiceAccount for nsx-node-agent
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: nsx-operator-svc-account
+  namespace: vmware-system-nsx
+---
+
+# Create ClusterRole for NCP
+kind: ClusterRole
+# Set the apiVersion to rbac.authorization.k8s.io/v1beta1 when k8s < v1.8
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: nsx-operator-cluster-role
+rules:
+  - apiGroups:
+      - ""
+      - extensions
+      - networking.k8s.io
+      - apps
+    resources:
+      - deployments
+      - endpoints
+      - pods/log
+      - nodes
+      - replicationcontrollers
+
+      # new resource for k8s 1.18
+      - ingressclasses
+
+
+    verbs:
+      - get
+      - watch
+      - list
+
+---
+
+# Create ClusterRole for NCP to edit resources
```

Comment:
do we need the role here? how about those roles defined in NCP AIO yaml?

Reply:
Only for local test, removed now.